spec/onceler_spec.rb

```diff
@@ -219,6 +219,46 @@ def my_method
     include_context "user cleanup"
   end
 
+  describe ".around(:once)" do
+    # don't use an ivar, they're not shared between arounds and examples
+    x = 0
+    around(:once) do |block|
+      x = 1
```

Comment:
might be good to say `x += 1` here and on line 233, so that you have more certainty that both blocks truly just run once